src/site/markdown/server/manage-guice-distributed-james.md

```diff
@@ -353,38 +352,38 @@ echo -e "\nauthorizer: org.apache.cassandra.auth.CassandraAuthorizer" >> /etc/ca
 
 #### Create a role
 
-Visit [cassandra documentation](http://cassandra.apache.org/doc/latest/cql/security.html) section `CREATE ROLE` for more information
+Have a look [cassandra documentation](http://cassandra.apache.org/doc/3.11.3/cql/security.html) section `CREATE ROLE` for more information
```

Comment:
No 3.11.4?

Reply:
I thought we're using `cassandra_3_11_3`, no?

Reply:
I just checked, you're right